src/core/acroform.cpp

```diff
@@ -0,0 +1,46 @@
+// SPDX-FileCopyrightText: 2022 James R. Barlow
+// SPDX-License-Identifier: MPL-2.0
+
+#include <qpdf/QPDF.hh>
+#include <qpdf/Constants.h>
+#include <qpdf/Types.h>
+#include <qpdf/DLL.h>
+#include <qpdf/QPDFExc.hh>
+
+#include <qpdf/QPDFObjectHandle.hh>
+#include <qpdf/QPDFAcroFormDocumentHelper.hh>
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "pikepdf.h"
+
+void init_acroform(py::module_ &m)
+{
+    py::class_<QPDFAcroFormDocumentHelper,
+        std::shared_ptr<QPDFAcroFormDocumentHelper>,
+        QPDFObjectHelper>(m, "AcroFormDocument")
```

Comment:
QPDFAcroFormDocumentHelper is a subclass of QPDFDocumentHelper, so the specification is simply

```
py::class_<QPDFAcroFormDocumentHelper,
        QPDFDocumentHelper>
```